src/SFA.DAS.EmployerIncentives.Data/IncentiveApplicationDataRepository.cs

```diff
@@ -19,5 +22,26 @@ public async Task Add(IncentiveApplicationModel incentiveApplication)
             await _dbContext.AddAsync(incentiveApplication.Map());
             await _dbContext.SaveChangesAsync();
         }
+
+        public async Task<IncentiveApplicationModel> Get(Guid incentiveApplicationId)
+        {
+            var incentiveApplication = _dbContext.Applications.AsNoTracking().FirstOrDefault(a => a.Id == incentiveApplicationId);
```

Comment:
If we're not tracking and have to update the object in the below method aren't we missing out on one of the bigger benefits of EF?

Reply:
I had to remove tracking so that the unit tests for update would work, happy to take suggestions on re-writing the tests to work with tracking

Reply:
I removed tracking because the unit tests were throwing the infamous "The instance of entity type 'Item' cannot be tracked because another instance with the same key value for {'Id'} is already being tracked" exception. A rewrite of the update code seems to have solved this.